tools/operator/testdata/barebones-test-input32.yml

```diff
@@ -37,7 +37,7 @@ invokes:
   - channelName: testorgschannel0
     name: samplecc
     targetPeers: OrgAnchor
-    nProcPerOrg: _NPROCPERORG_
+    nProcPerOrg: 32
     nRequest: 10000
```

Comment:
instead of 10000, can this be reduced to a lower number as we already stressing the peer with 32 concurrent threads and to make test run shorter. Currently it is taking around 1 hour 45 minutes to complete 32 and 36 concurrent invokes.

Reply:
if we think 1.75 hr is too long, then we can just do one test, 32 threads.  We can add more test back if needed. ok?

Reply:
we can run both but do we need 10000 transactions on each thread? 

Can we lower 10000 transactions per thread to 1000 transactions per thread?

Reply:
For performance runs, a relative large number transactions are needed to be a good representation though the TPSs may not differ significantly among them.

36 threads run does not give us additional more information.  Let's do 1 run (32 threads) for 10k per thread, ok?

Reply:
yes, agree.